script/check_publish.sh

```diff
@@ -12,8 +12,8 @@ source "$SCRIPT_DIR/common.sh"
 
 function check_publish() {
   local failures=()
-  for package_name in "$@"; do
-    local dir="$REPO_DIR/packages/$package_name"
+  for dir in $(pub global run flutter_plugin_tools list --plugins="$1"); do
```

Comment:
Seems wrong to only look at the first argument (prior to this change it would check the packages for all arguments)

Reply:
That was because previously, `check_publish` was called with the dereferenced `CHANGED_PACKAGE_LIST` array, which passed the array items as separate arguments. Now there is only one argument, the packages list in a comma-separated list.